packages/light-apps/package.json

```diff
@@ -41,6 +41,8 @@
     "semantic-ui-css": "^2.4.1",
     "store": "^2.0.12",
     "symbol-observable": "^1.2.0",
+    "tachyons": "^4.11.1",
+    "tachyons-components": "^1.0.1",
```

Comment:
Since ui-components has `tachyons-components`, and light-apps includes ui-components, I think we can remove `tachyons-components` from this package.json (only leave it in ui-components)